internal/resource/sysfs-lib.go

```diff
@@ -21,11 +21,18 @@ import (
 	"k8s.io/klog/v2"
 )
 
-type vfioLib struct{}
+type vfioLib struct {
+	nvpci.Interface
```

Comment:
Do we really want to expose the entire `nvpci.Interface` here? can we use:
```
type vfioLib struct {
    nvpcilib nvpci.Interface
}
```
instead to ensure that we don't do so?

Reply:
Fixed